cmd/mount2/mount_test.go

```diff
@@ -1,16 +1,14 @@
-//go:build linux || (darwin && amd64)
-// +build linux darwin,amd64
+//go:build linux
+// +build linux
```

Comment:
I limited this test to linux since it was resulting in lockups on the macos runner. Let me know if you want me to revert this change.

Reply:
That is a good idea - thanks.